providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -317,6 +327,30 @@ static const OSSL_PARAM *dh_gettable_params(void)
     return dh_params;
 }
 
+static const OSSL_PARAM dh_known_settable_params[] = {
+    OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_TLS_ENCODED_PT, NULL, 0),
+    OSSL_PARAM_END
+};
+
+static
+const OSSL_PARAM *dh_settable_params(void)
```

Comment:
Line break?
Same for the function below.

Reply:
Fixed